package.json

```
@@ -47,12 +48,9 @@
       "eslint"
     ]
   },
-  "engines": {
-    "yarn": "^1.21.1"
-  },
   "resolutions": {
     "@typescript-eslint/eslint-plugin": "^7.18.0",
     "@typescript-eslint/parser": "^7.18.0"
   },
-  "packageManager": "yarn@1.22.19"
+  "packageManager": "yarn@4.5.2+sha512.570504f67349ef26d2d86a768dc5ec976ead977aa086b0bb4237e97d5db7ae5c620f9f0e0edf3ea5047205063faff102bf2a2d778664a94eaaa1085ad483fe2e"
```

```suggestion
  "packageManager": "yarn@4.5.2"
```
автоматом зарезолвил с хешом

> PS: или PR ещё в драфте?

---

Фича, хэш все равно будет при corepack up

Не в драфте